lambdas/functions/management/management.go

```diff
@@ -0,0 +1,59 @@
+package management
+
+import (
+	"context"
+	"fmt"
+	"github.com/weka/aws-tf/modules/deploy_weka/lambdas/functions/weka_api"
+
+	"github.com/weka/go-cloud-lib/connectors"
+	"github.com/weka/go-cloud-lib/lib/jrpc"
+	"github.com/weka/go-cloud-lib/lib/weka"
+	"github.com/weka/go-cloud-lib/logging"
+	"github.com/weka/go-cloud-lib/protocol"
+)
+
+func GetWekaJRPC(ctx context.Context, request weka_api.WekaApiRequest) (interface{}, error) {
+	logger := logging.LoggerFromCtx(ctx)
+	logger.Debug().Msg("GetWekaJRPC > Start")
+
+	var method weka.JrpcMethod
+	var jrpcProtocol interface{}
+	switch request.Method {
+	case "status":
```

Comment:
no need for special handling for the status here, should be done on the status call, we anyway return interface

Reply:
Here I need to switch the method type so I know which `protocol` struct I send to be filled by the JRPC call. If I send an `interface{}` the data gets marshaled according to what's returned, including fields which are not part of the existing `protocl.WekaStatus{}` struct

Reply:
management is generic, should support any JRPC call, we can't list all of them, the actual response struct should be defined outside in the caller

Reply:
You mean to define the `protocol.WekaStatus` struct in the `statusHandler` function which calls WekaApi which calls management?

Reply:
I will still need a `switch` statement to match between the Method (string) and the `weka.jrpcMethod` enum that is passed to the `jpool.Call` method

Reply:
827458a77e41df0454fc12a18341640b307352bd

I return `json.RawMessage` instead, and for the `method` - I check it is supported when calling `wekaApi`